frontend/src/components/videoPlayer/Comments.js

```diff
@@ -28,79 +27,43 @@ const useStyles = makeStyles((theme) => ({
   },
 }));
 
-const Comments = () => {
+const Comments = ({ movie }) => {
   const classes = useStyles();
 
+  if (!movie || !movie.comments) return <div></div>;
+
   return (
+    <>
     <div className={classes.root}>
       <List>
-      <ListItem alignItems="flex-start">
-        <ListItemAvatar>
-          <Avatar alt="Remy Sharp" src="https://images.unsplash.com/photo-1613053341085-db794820ce43?ixid=MXwxMjA3fDB8MHxlZGl0b3JpYWwtZmVlZHwxMHx8fGVufDB8fHw%3D&ixlib=rb-1.2.1&auto=format&fit=crop&w=500&q=60" />
-        </ListItemAvatar>
-        <ListItemText
-          primary="Ali Connors"
-          secondary={
-              <Typography
-                component="span"
-                variant="body2"
-                className={classes.inline}
-                color="textPrimary"
-              >
-                FIRST!!!!!!!!!111111 Fu**** awesome movie! Bruce rocks!
-              </Typography>
-          }
-        />
-      </ListItem>
-      <Divider className={classes.dividerStyle} variant="inset" component="li" />
-      <ListItem alignItems="flex-start">
+      {movie.comments.map((e, index) => (
```

Comment:
This works perfectly, but is a bit hard to read:
- since you are mapping comments, `c` would be more understandable abbreviation than `e`
- `userId` is actually the whole `user` so it should be named that way
- `index` should not be used as a key. Since we are not removing them this should work, but it would be safer if comments had ids to be used as keys